browser/net/brave_ads_status_header_network_delegate_helper_unittest.cc

```diff
@@ -66,6 +67,7 @@ class AdsStatusHeaderDelegateHelperTest : public testing::Test {
   }
 
  private:
+  brave_l10n::test::ScopedDefaultLocale scoped_locale_{"en_US"};
```

Comment:
Some tests always fail on my machine because of banned locale https://github.com/brave/brave-core/blob/6400c1366485fede824438d95ba70994433e9380/components/l10n/common/ofac_sanctioned_iso_3166_1_country_code_constants.h#L32